lib/cisco_node_utils/vxlan_vtep.rb

```diff
@@ -35,7 +35,7 @@ def initialize(name, instantiate=true)
     def self.vteps
       hash = {}
       is_vxlan_feature = config_get('vxlan', 'feature')
-      return hash if (:enabled != is_vxlan_feature.first.to_sym)
+      return hash if (:enabled != is_vxlan_feature.to_sym)
```

Comment:
I don't understand the need for :enabled / :disabled ? Our other feature tests usually do something like this:

```
     def self.vteps
       hash = {}
       return hash unless feature.enabled
       ...
    end

    def self.enabled
      config_get('vxlan', 'feature')
    rescue Cisco::CliError => e
      # cmd will syntax reject when feature is not enabled
      raise unless e.clierror =~ /Syntax error/
      return false
    end
```